liquigraph-core/src/main/java/org/liquigraph/core/configuration/ConfigurationBuilder.java

```diff
@@ -98,6 +102,15 @@ public ConfigurationBuilder withUsername(String username) {
     }
 
     /**
+     * Specifies the default username as the username allowed to connect to the remote
+     * graph database instance.
+     * @return itself for chaining purposes
+     */
+    public ConfigurationBuilder withDefaultUsername() {
```

Comment:
As can be seen in #130, my initial interest was just in avoiding the meaningless exception, so I'm not overly inclined to introduce this `.withDefaultUser()` method at all, especially regarding Neo4j introducing multi-user (and not knowing how long the default user will be available). IMHO this can be undone, but maybe @fbiville wants to add a few thoughts on that, since initially this was his idea. Maybe I'm (or we both are) overlooking some detail here?!

Keeping the API simpler is a strong argument, too.